modules/apps/static/portal-k8s-agent/portal-k8s-agent-impl/src/main/java/com/liferay/portal/k8s/agent/internal/osgi/commands/ClientExtensionsOSGiCommands.java

```diff
@@ -276,13 +287,13 @@ private Configuration _reloadConfiguration(Configuration configuration)
 				InMemoryOnlyConfigurationThreadLocal.
 					setInMemoryOnlyWithSafeCloseable(true)) {
 
-			Configuration reloadedCxConfiguration =
+			Configuration reloadedConfiguration =
```

Comment:
here it is `reloadedConfiguration`, but above it is `reloadedClientExtensionConfiguration`. Let's go with `reloadedConfiguration`.